Gemfile

```diff
@@ -60,3 +60,5 @@ group :production do
   # Webserver
   gem 'puma'
 end
+
+gem "twitter-bootstrap-rails"
```

Comment:
Nunca me convenci de uma real utilidade no `less` comparado ao `sass` em projetos Rails, quem sabe se for convencido, fiquei curioso pela sua escolha... Por mais pessoal que seja, você deve ter algum (ou alguns?) motivo que o levou a trocar um pelo outro. (: